### PR TITLE
fix: adding http/https options to readinessProbes for console

### PR DESCRIPTION
**Pull request by hamza-m-masood**
### Which problem does the PR fix?

closes: https://github.com/camunda/camunda-platform-helm/issues/2376

### What's in this PR?

I used if statements to check the scheme of each component's readinessProbe. Then configured console to use that scheme for it's own readiness checks on all other components.

I could not use the `ternary` operator because it only accepts booleans.

### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/docs/contributing.md).

**Before opening the PR:**

- [ ] In the repo's root dir, run `make go.update-golden-only`.
- [ ] There is no other open [pull request](../pulls) for the same update/change.
- [ ] Tests for charts are added (if needed).
- [ ] In-repo [documentation](../blob/main/docs/contributing.md#documentation) are updated (if needed).

**After opening the PR:**

- [ ] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

File: charts/camunda-platform-8.6/templates/camunda/_helpers.tpl [hunk 1]
```diff
@@ -561,7 +561,11 @@ Release templates.
   {{- $baseURL := printf "%s://%s" $proto .Values.global.ingress.host }}
 
   {{- if .Values.console.enabled }}
-  {{- $baseURLInternal := printf "http://%s.%s:%v" (include "console.fullname" .) .Release.Namespace .Values.console.service.managementPort }}
+  {{- $proto := "http" -}}
+  {{- if eq (lower .Values.console.readinessProbe.scheme) "https" -}}
```
Comment (aabouzaid) [inline charts/camunda-platform-8.6/templates/camunda/_helpers.tpl:565]: There is no need to make if statement, just use the scheme directly.

```
{{-  $proto := (lower .Values.console.readinessProbe.scheme) -}}
```

File: charts/camunda-platform-8.6/templates/camunda/_helpers.tpl [hunk 2]
```diff
@@ -561,7 +561,11 @@ Release templates.
   {{- $baseURL := printf "%s://%s" $proto .Values.global.ingress.host }}
 
   {{- if .Values.console.enabled }}
-  {{- $baseURLInternal := printf "http://%s.%s:%v" (include "console.fullname" .) .Release.Namespace .Values.console.service.managementPort }}
```
Comment (aabouzaid) [inline charts/camunda-platform-8.6/templates/camunda/_helpers.tpl:564]: The changes should be in the Alpha chart.

**Review by aabouzaid — CHANGES_REQUESTED** [review]

File: charts/camunda-platform-8.6/templates/camunda/_helpers.tpl (hunk 1 above)
Comment (hamza-m-masood) [reply charts/camunda-platform-8.6/templates/camunda/_helpers.tpl]: done

File: charts/camunda-platform-8.6/templates/camunda/_helpers.tpl (hunk 2 above)
Comment (hamza-m-masood) [reply charts/camunda-platform-8.6/templates/camunda/_helpers.tpl]: done

**Review by aabouzaid — APPROVED** [review]
LGTM, thank you 👍
Please don't forget to update the golden files and unit test if needed.

**Comment by hamza-m-masood** [comment]
updated the golden files and readme locally. There is no change.

**Pull request merged**